src/components/applications/ApplicationList.js

```diff
@@ -31,14 +23,7 @@ const ApplicationList = (props) => {
       ) : (
         <Datagrid rowClick="show">
           <TextField label="Name" source="applicationName" />
-          <ReferenceField
-            link="show"
-            label="Organization"
-            source="orgId"
-            reference="Organizations"
-          >
-            <TextField label="Organization Name" source="organizationName" />
-          </ReferenceField>
```

Comment:
What was the reasoning behind this change?

Reply:
many thanks @simply-alliv I explained in PR description - background context..